### ISSUE TITLE
Vehicles auto-braking on high fps

### ISSUE THREAD

**Issue by Xenius97** [multitheftauto/mtasa-blue#2935]
### Describe the bug

If you release acceleration btn your car will braking automatically
https://streamable.com/xs935s

### Steps to reproduce

Get a car
https://github.com/multitheftauto/mtasa-resources/issues/419

### Version

All

### Additional context

Parent issue:
* #929

**Comment by TracerDS** [comment]
https://wiki.multitheftauto.com/wiki/SetFPSLimit

<h4>74 FPS is the breaking point that opens the door to various more severe GTA bugs related to FPS and physics.</h4>

* Physics of vehicles is effected, both high and low FPSes may bring their own set of unfair advantages. Speaking about the consequences of high FPS in this context, up to 70 or 74 FPS is considered safe (as any differences in physics, if they do exist to begin with as they theoretically should, are so tiny that they are unmeasurable and thus wouldn't affect racing results in practise). Anything beyond 74 FPS may cause impactful discrepancies.
* Pressing the horn button to turn on and off sirens gets really hard at very high FPS. For instance, at 100 FPS, you are more likely to hit the regular horn 3 times (inconsistent) before eventually triggering the siren, besides taking a similar amount of tries to turn off the siren.
* At very high FPS, climbing over certain objects will result in instant death. Example at: 2520.108, -1681.407, 19.406, 266
* The higher your FPS, the more of a penalty in satchel throwing distance (up to ~10% at very high FPS) will apply.

TLDR; The higher the FPS, the worse the gameplay. The same thing happens in singleplayer at FPS >= 60

**Comment by Xenius97** [comment]
> #### 74 FPS is the breaking point that opens the door to various more severe GTA bugs related to FPS and physics.

But still a bug, many fps bugs got fixed a few months ago.
I've noticed this above 150fps (https://streamable.com/jo9e43)

**Comment by TracerDS** [comment]
iirc this is the engine's issue. Not much can be fixed with shitty engine. Only small patches. If it wasnt fixed for a very long time, that means it cant be fixed without changing the source code of the gta itself (otherwise someone would have already done it, right?)

**Comment by botder** [comment]
A branch in my repository fixes some problems (partially; while introducing others) including the "auto-braking on high fps": https://github.com/botder/mtasa-blue/commits/feature/fixed-physics-rate

**Comment by lopezloo** [comment]
Parent issue:
* #929

(have edited first post to include that)

**Comment by lopezloo** [comment]
This issue was fixed in 7c8a1ab93d091b06262749b482d6c8142ca69eea (#2784), then reverted in ef792d6af62443f97014621334c7188dddb4ef29 (#2787) due to breaking compatibility on race maps.

Thus could be readded as optional switch in [`setWorldSpecialPropertyEnabled`](https://wiki.multitheftauto.com/wiki/SetWorldSpecialPropertyEnabled) or [`setGlitchEnabled`](https://wiki.multitheftauto.com/wiki/SetGlitchEnabled) function.

@Merlin

**Comment by Yok009** [comment]
give me scripts

**Comment by FileEX** [comment]
Fixed in https://github.com/multitheftauto/mtasa-blue/commit/3f5801e65d8a51d112b686485d4a2491151c3311
